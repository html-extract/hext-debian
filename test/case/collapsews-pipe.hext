<article @strip-tags:collapsews:t />
