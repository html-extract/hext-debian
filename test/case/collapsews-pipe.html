<article>
  <h1>Something is happening!</h1>
  <p>And <b>you</b> don't know <i>what</i> it is</p>
</article>
