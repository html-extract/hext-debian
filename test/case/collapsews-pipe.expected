{"t":"Something is happening! And you don't know what it is"}
